site/src/pages/docs/server-annotated-service.mdx

```diff
@@ -753,6 +753,52 @@ public class MyAnnotatedService {
 }
 ```
 
+## Specifying a blocking task executor
+
+An annotated service is executed by an [EventLoop](https://netty.io/4.1/api/io/netty/channel/EventLoop.html),
```

Comment:
I'm worried that this page is already too long. 
Is it better to create a dedicated page for the event loop and blocking IO thread?
The page could contain:
- Eventloop overview
- What does blocking mean?
- How to execute a blocking IO in:
  - Annotated service
  	- `@Blocking`
  	- `ctx.blockingTaskExecutor()`
  - Thrift service using `AsyncIface` and `Iface`
  - gRPC service using
    - GrpcServiceBuilder.useBlockingTaskExecutor(boolean)`
    - `ServiceRequestContext.current().blockingTaskExecutor()`
- Configure your own blocking task executor with `ServerBuilder`

Reply:
> I'm worried that this page is already too long.

Yes, that's why I didn't explain about `EventLoop` and other stuff but focusing on `@Blocking`.

> Is it better to create a dedicated page for the event loop and blocking IO thread?

Yeah, I guess we need that. I was trying to find another reference as I asked before https://github.com/line/armeria/pull/3390/files#r592997369
but I guess we need to add to our documentation.

Reply:
Also, I think we still have to contain the usage of `@Blocking` here and link to the page instead of completely moving there. What do you think?

Reply:
>  I think we still have to contain the usage of @Blocking here and link to the page instead of completely moving there.

Agreed.